tap_square/client.py

```diff
@@ -311,6 +312,14 @@ def get_payments(self, start_time, bookmarked_cursor):
 
             yield (result.body.get('payments', []), result.body.get('cursor'))
 
+    def get_batch_token(self, link): #pylint: disable=no-self-use
+        if link:
+            url = link[link.find('<')+1:link.find('>')]
+            parsed = urllib.parse.urlparse(url)
+            batch_token = urllib.parse.parse_qs(parsed.query)['batch_token'][0]
+            return int(batch_token)
+        return None
```

Comment:
Don't we already have this method used elsewhere?

Reply:
Removed